controllers/recipes.js

```diff
@@ -0,0 +1,170 @@
+const Recipe = require("../models/Recipe");
+const Review = require("../models/Review");
+const Comments = require("../models/Comments");
+const User = require("../models/User");
+
+module.exports = {
+  /* CREATE */
+
+  createRecipe: async (request, response) => {
+    const {
+      title,
+      picturePath,
+      servings,
+      steps,
+      ingredients,
+      time,
+      userId,
+      tags,
+    } = request.body;
+
+    const recipe = new Recipe({
+      title,
+      picturePath,
+      servings,
+      steps,
+      ingredients,
+      time,
+      userId,
+      tags,
+    });
+
+    const user = request.user;
+    if (!user) {
+      return responseponse.status(401).json({error: "operation not permitted"});
+    }
+
+    recipe.userId = user._id;
+
+    const createdRecipe = await recipe.save();
+    user.recipes = user.recipes.concat(createdRecipe._id);
+    response.status(201).json(createdRecipe);
```

Comment:
You could create some constants for the status codes, instead of hardcoding the number. It's better readability since it can be hard to recall what each number represents.

eg. `STATUS_CODE.CREATED` instead of 201

Reply:
Would I put this in a shared file? 
I know typescript does this not sure about javascript

Reply:
Yes, you can add it in a share file

Reply:
dumb question. Do I just create an object?
const status = 
{
STATUS_CODE.CREATED: '404'
}

and it would just work?

Reply:
Something like this?
```
const STATUS_CODE = {
  CREATED: 201,
  NOT_FOUND: 404,
};
```